content/knowledge_base/glossary.md

```diff
@@ -0,0 +1,141 @@
+# Glossary
+
+## General Blockchain Concepts
+
+### Consensus
+
+TODO: add
+
+### Blockchain
+
+TODO: add
+
+### Gossip Algorithms
+
+TODO: add
+
+### Liveness Threshold
+
+The number of malicious participants that can be tolerated before the consensus protocol stops functioning.
+
+### Nakamoto Type of Consensus
+
+TODO: introduce (for BTC)
+A consensus algorithm that provides a probabilistic safety guarantee; its decisions may revet with some probability ε.
```

Comment:
I guess you meant

```suggestion
A consensus algorithm that provides a probabilistic safety guarantee; its decisions may revert with some probability `ε`.
```

? Also when the variable is not used later in this context, I'm not sure that it makes sense to say `with some probability ε`. Do you plan on using that variable later on?

Reply:
Generally yes, if there are any technical parts, but I do not have anything specific in mind.

Reply:
Corrected https://github.com/optakt/optakt.github.io/pull/13/commits/48a7f92c1e451ec49d7b7f2cb79fa55a85a83a48#diff-55a59bbede67c988e24e88ffb8a7b56d20454b9ab95a3fd8567e790585338dfbR24
Thanks!